tests/queries/0_stateless/03036_clamp.sql

```diff
@@ -0,0 +1,8 @@
+SELECT clamp(1, 10, 20);
```

Comment:
with NaN and Null tests, please test that max boundary is also respected and not just min

Reply:
@antonio2368 Hi, I am not very familiar with boundary testing for NaN and Null. Can you provide an example, please?

Reply:
I think it will be enough to add NULL as values/boundaries, same for NaN.
You can verify the behavior to be consistent with `least(greatest(x, min), max)`

Reply:
@antonio2368 The results of greatest(nan, 1) and greatest(1, nan) are inconsistent. Does the clamp function also need to behave this way? Also, I have a question: are NaN and null considered greater or less than any value?

Reply:
@antonio2368 Can you help me answer my questions?

Reply:
Anyhting compared to NULL should give NULL in the end.

For NaN, yes it seems a bit confusing, but I think it's okay to just leave it as it is.